tests/unit/style/parsing/mod.rs

```diff
@@ -81,6 +81,7 @@ mod animation;
 mod background;
 mod basic_shape;
 mod border;
+mod box_shadow;
```

Comment:
It doesn't seem to me you need a new mod for this test at all. Could you just add the test into `effects.rs`?

You may also add a test to `inherited_text.rs` for `text-shadow`.